README.md

```diff
@@ -151,8 +151,11 @@ All output from `go-java-launcher` itself, and from the launch of all processes
 By default, when starting a java process inside a container (as indicated by the presence of ``CONTAINER`` env
 variable):
 
+1. `-XX:ActiveProcessorCount` will be set based on the discovered cgroup configurations and host information to a value
+   between 2 and the number of processors reported by the runtime. You can read more about the reasoning behind this
+   [here](https://github.com/palantir/go-java-launcher/issues/313).
```

Comment:
Worth mentioning that if a `-XX:ActiveProcessorCount` value is already set, we don't modify it?